src/app/auth/singup/singup.component.html

```diff
@@ -0,0 +1,34 @@
+<div class="login-container">
+  <app-window icon="🔑" label="Sign Up">
+    <div class="login-window-pane">
+      <div class="under-ridged logo-container">
+        <div class="logo-icon">🏃</div>
+        <div class="logo-label">Angular Instant Messenger</div>
+      </div>
+      <form class="form" [formGroup]="signUpForm" (ngSubmit)="signup()">
+        <label for="username"> Username🔑 </label>
+        <input required id="username" formControlName="username" />
```

Comment:
It can also be a good idea to add validation attributes here.
maxlength="32" minlength="3"